src/Styles/Details.css

```diff
@@ -0,0 +1,39 @@
+.characters {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  color: whitesmoke;
+}
+
+.characters > div:nth-child(even) {
+  text-align: center;
+  padding: 10px 0;
+  width: 100%;
+  background-color: rgba(41, 79, 156, 0.921);
+}
+
+.characters > div:nth-child(odd) {
```

Comment:
- [ ] [REQUIRED] Good job so far! You did well all your project but you need to give this style(`:nth-child(odd)`) to also `Home` page elements, it should looks provided as [Behance](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) 🙏🏼 

![image](https://github.com/Pi1998/marvel-folio/assets/63915024/9ce568ad-02fc-488d-8e15-ca452f35a968)